get_involved.md

```diff
@@ -20,25 +22,27 @@ There are plenty of opportunities for you to contribute to the HSF, be it by
 participation in technical discussions, by joining one of our working groups, or
 by joining the HSF with your software project:
 
-  *  Participation in HSF activities and forums. See the Activities page.
-  *  Participation in HSF events and meetings.
-  *  Contributing to the [HEP S&C Knowledge Base](http://hepsoftware.org)
-  *  Taking advantage for your project from the HSF (to be created)
-  *  Giving input to the HSF startup team
+  *  Participation in HSF [working groups](/what_are_WGs.html) and activities
+  *  Participation in [HSF events and meetings](/future-events.html)
+  *  Taking advantage of the HSF by indentifying your [software project](/projects.html) with us
+  *  Giving input to the [HSF Coordination Team](/organization/team.html)
   *  Contributing to this website
 
 ## Meetings, Agendas and Video-Conferences Access
 
-See the Events listing on this site for information on HSF meetings, workshops,
-etc. Meetings use the [CERN Indico system](http://indico.cern.ch/category/5816/) for agenda pages. The agenda pages are open to the world, and no CERN account needed. The Vidyo video conferencing system is used for all HSF meetings and anyone can participate in the meetings from any internet-connected location. 
-Connection information for each specific Vidyo meeting is on the corresponding Indico meeting agenda page.  Vidyo has a web-browser interface, or alternatively one can [download the Vidyo desktop client (standalone application)](http://information-technology.web.cern.ch/services/fe/howto/users-install-vidyo-desktop-client).  However should you need to upload material to the
-agenda, a CERN account is needed. CERN provides lightweight accounts to non CERN
-affiliated people when needed. See <https://account.cern.ch/account/Externals/>
+See the [Events listing](/future-events.html) on this site for information on HSF meetings, workshops,
+etc. 
+
+Meetings use the [CERN Indico system](http://indico.cern.ch/category/5816/) for agenda pages. The agenda pages are generally open to the world, and no CERN account needed. 
+
+The [Vidyo](http://information-technology.web.cern.ch/services/fe/vidyo) video conferencing system is used for all HSF meetings and anyone can participate in the meetings from any internet-connected location. Connection information for each specific Vidyo meeting is on the corresponding Indico meeting agenda page.
+(To upload material to Indico you do need to be identified, in addition to a CERN computing account CERN now allows EduGain authentication
```

Comment:
This sentence does not read well. It seems it should be 2 sentences.

BTW, one can also upload to Indico with a lightweight accout? Worth mentioning if that's the case.

Reply:
Lightweight accounts are being phased out in favour of eduGain though

Reply:
OK OK.